web/index.html

```diff
@@ -1463,19 +1474,24 @@ <h4 class="modal-title" id="switchRegistryModalLabel">Switch Netdata Registry Id
 	},
 
 	'disk.iotime': {
-		height: 0.5
+		height: 0.5,
+		info: 'Number of milliseconds spend by all operation. Read from fields 4 and 8.'
 	},
 	'disk.mops': {
-		height: 0.5
+		height: 0.5,
+		info: 'Number of merged operations. Read from fields 2 and 6.'
 	},
 	'disk.svctm': {
-		height: 0.5
+		height: 0.5,
+		info: 'Buisy time devided through the number of operations.'
```

Comment:
The average service time (in milliseconds) for completed I/O operations. This metric is calculated using the total busy time of the disk and number of completed operations. If the disk is able to execute multiple parallel operations the reporting average service time will be misleading.